static/css/learnocaml_description.css

```diff
@@ -52,14 +53,97 @@ body {
   overflow: auto;
   flex: 1 3 auto ;
 }
-#learnocaml-exo-tab-meta > h1::after {
+#learnocaml-exo-tabQ > * > h1::after {
```

Comment:
Hi @YoanwM, thanks!
I just read your PR code (not tested yet) and it looks quite good.
Just one question for the moment: is this change expected?
(`#learnocaml-exo-tab-meta` → `#learnocaml-exo-tabQ`)

Reply:
I can't remember I did this, I think it's just a mistake I didn't see !